### ISSUE TITLE
Option to turn off borders

### ISSUE THREAD

**Issue by IndoJimbo** [spiritualized1997/openFPGA-Super-GB#2]
Can you add an option to turn off borders so we can play the games full screen?

**Comment by DFXThomas** [comment]
Yes, the Super Gameboy should be displayed using square pixels, not 4:3 or anything else.

+1

**Comment by Social0** [comment]
Yep, a big +1 from me as well, that would be awesome. Too small to play currently, when not docked.

**Comment by NinjaNick125** [comment]
Another +1 to this request. I would love to have the option to simply zoom in to the same aspect ratio and resolution as the game boy so we can play these games in their best way, with the best picture quality.
